Sources/REventLogger/Extensions/Device+DeviceMetaData.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import UIKit
+
+@objc public protocol EventLoggerDeviceable {
+    var platform: String? { get }
+    var osVersion: String? { get }
+    var deviceModel: String? { get }
+    var deviceBrand: String? { get }
+    var deviceName: String? { get }
+}
+
+extension UIDevice: EventLoggerDeviceable {
+    public var platform: String? {
+        systemName
```

Comment:
This returns a `non-optional String`. Any reason to make `platform` an `Optional String`?